python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -462,35 +477,67 @@ async def run_cwfs(self):
         results_dict = self.calculate_results()
         return results_dict
 
+    def get_donut_region(self, center):
+
+        center_y = center[0]
+        center_x = center[1]
+        return (
+            center_y - self.side,
+            center_y + self.side,
+            center_x - self.side,
+            center_x + self.side,
+        )
+
+    def get_intra_donut_center(self):
+        return [
+            int(self.intra_result.brightestObjCentroidCofM[1]),
+            int(self.intra_result.brightestObjCentroidCofM[0]),
+        ]
+
+    def get_extra_donut_center(self):
```

Comment:
Just to clarify, you can make this method something like this:

```
    def get_extra_donut_center(self):
        return (
            int(self.extra_result.brightestObjCentroidCofM[1]),
            int(self.extra_result.brightestObjCentroidCofM[0])
            if not self.extra_focal_position_out_of_range
            else self.get_intra_donut_center(),
        )
```

Reply:
hmm, i commented on this but must have not hit "Comment"
Implemented, but not as a single line... it was too hard to read.